src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal.tsx

```diff
@@ -86,10 +85,11 @@ const CreateNewListModal: React.FC<CreateNewListModalProps> = ({
       onComplete()
     } catch (error) {
       logger.error(error)
-
-      setFormFieldError(
-        error.message ?? t("collectorSaves.createNewListModal.genericError")
-      )
+      helpers.setSubmitting(false)
```

Comment:
`isSubmitting` will be automatically set to `false` when promise finishes executing